catalog/imaging/imaging-transformer-chipping/pom.xml

```diff
@@ -101,13 +150,69 @@
                         <!-- Don't Export Anything -->
                         <Export-Package/>
+                        <Import-Package>
+                            !org.codice.alliance.catalog.core.api.impl.types,
+                            !sun.security.action,
+                            *
+                        </Import-Package>
                         <Embed-Dependency>
-                            catalog-core-api-impl,
-                            platform-util
+                            catalog-core-api-impl;groupId=org.codice.alliance.catalog.core;version=${project.version},
+                            catalog-core-api-impl;groupId=ddf.catalog.core;version${ddf.version},
+                            platform-util,
+                            imaging-service-impl,
+                            commons-lang3,
+                            codice-imaging-nitf-core,
+                            codice-imaging-nitf-render,
+                            codice-imaging-nitf-fluent-api,
+                            la4j,
+                            jai-imageio-jpeg2000,
+                            jai-imageio-core
                         </Embed-Dependency>
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>true</haltOnFailure>
+                            <rules>
+                                <rule>
+                                    <element>BUNDLE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>INSTRUCTION</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.87</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>BRANCH</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.56</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>COMPLEXITY</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.71</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>LINE</counter>
```

Comment:
This commit removes the line coverage thresholds from Alliance, so this is no longer needed.
https://github.com/codice/alliance/commit/242f4e2566be355b2aa99e31e886348920c896a7